sympy/stats/crv_types.py

```diff
@@ -1972,6 +2023,11 @@ class NakagamiDistribution(SingleContinuousDistribution):
 
     set = Interval(0, oo)
 
+    @staticmethod
+    def check(mu, omega):
+        _value_check(mu >= 0.5, "mu, shape parameter should be real.")
```

Comment:
test doesn't match message: mu.is_real or "mu should be real and not less than 1/2"